art/estimators/classification/pytorch.py

```diff
@@ -261,12 +261,12 @@ def fit_generator(self, generator: "DataGenerator", nb_epochs: int = 20, **kwarg
             for _ in range(nb_epochs):
                 for i_batch, o_batch in generator.iterator:
                     if isinstance(i_batch, np.ndarray):
-                        i_batch = torch.from_numpy(i_batch).to(self._device)
+                        i_batch = torch.from_numpy(i_batch).type(torch.FloatTensor).to(self._device)
```

Comment:
What is the reason for this and similar additions in lines 269, 318, 396, 404, and 496?

Reply:
Our modifications are neglectable. These were just for temporary measures. I have to push the corrected version (i.e., the original version in IBM:dev_1.4.0)?